Source/WebCore/rendering/RenderMenuList.cpp

```diff
@@ -327,6 +327,9 @@ LayoutRect RenderMenuList::controlClipRect(const LayoutPoint& additionalOffset)
 
 void RenderMenuList::computeIntrinsicLogicalWidths(LayoutUnit& minLogicalWidth, LayoutUnit& maxLogicalWidth) const
 {
+    if (style().fieldSizing() == FieldSizing::Content)
+        return RenderFlexibleBox::computeIntrinsicLogicalWidths(minLogicalWidth, maxLogicalWidth);
+
```

Comment:
how does this interact with size containment?

Reply:
That's a good catch I hadn't thought about that properly (I saw RenderFlexibleBox handles containment in some fashion but didn't think further).

contain: size; with field-sizing: content; currently breaks the select. It similarly makes the select nearly unusabl within Chromium so I think this is a deeper issue. I will make a follow up to investigate this and see if it needs spec work or at least what the expectation is.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=269169

Reply:
👍 please add a FIXME referencing this bugzilla.